src/Desktop.Analyzers/Core/Helpers/SecurityDiagnosticHelpers.cs

```diff
@@ -164,6 +165,30 @@ public static int HasXmlReaderParameter(IMethodSymbol method, CompilationSecurit
             return HasSpecifiedParameter(method, xmlTypes, IsXmlReaderType);
         }
 
+        public static bool IsExpressionEqualsNull(IOperation operation)
+        {
+            ILiteralExpression literal = operation as ILiteralExpression;
+            return literal != null && literal.ConstantValue.HasValue && literal.ConstantValue.Value == null; 
+        }
+
+        public static bool IsExpressionEqualsDtdProcessingParse(IOperation operation)
+        {
+            IFieldReferenceExpression enumRef = operation as IFieldReferenceExpression;
+            return enumRef != null && enumRef.ConstantValue.HasValue && (int)enumRef.ConstantValue.Value == 2; // DtdProcessing.Parse
```

Comment:
You have the `DtdProcessing` type under SecurityTypes. Can you verify if the value refers to the Field Symbol `DtdProcessing.Parse`? I am not really sure how feasible that is here